10_transformers-from-scratch.ipynb

```diff
@@ -193,6 +193,13 @@
     "#### Memory mapping"
```

Comment:
<p>Suggestion as above:</p><p> </p><blockquote><strong>Note</strong>: In the following code block, a large GPT-2 checkpoint is loaded into memory. On platforms like Colab and Kaggle, this can cause the instance to crash due to insufficient RAM or GPU memory. You can still run the example if you replace the large model with the small one by using<code>model_ckpt = "transformersbook/codeparrot-small"</code>.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>